tracing/trace_keyspace_helper.cc

```diff
@@ -87,6 +87,7 @@ trace_keyspace_helper::trace_keyspace_helper(tracing& tr)
                                   "started_at timestamp,"
                                   "request_size int,"
                                   "response_size int,"
+                                  "username text,"
```

Comment:
Will this work after upgrading an existing cluster? There's a "IF NOT EXISTS" in line 79 so I would expect it won't. Please run a version of Scylla without your change and use tracing on it. Then start a version of Scylla with your change and verify everything works as expected and the new field is there and filled correctly.

Reply:
It doesn't work with upgrading as it is right now. I don't see how we can remember the usernames for requests before upgrading, should this field be filled with nulls for them?

Reply:
Filling the new column up with nulls for old entries is alright, I think.

Reply:
You'll have a race between preparing the statement and altering the table to add username. So I suggest to try to prepare with username, and fall back to without username. There's a chance that executing a query with an extra parameter won't hurt.